src/components/sankey-chart/sankey-focus-manager.ts

```diff
@@ -0,0 +1,124 @@
+import { DOWN_ARROW, LEFT_ARROW, RIGHT_ARROW, UP_ARROW } from '@angular/cdk/keycodes';
```

Comment:
Perhaps in future (not this sprint) we can make a general purpose 2D focus manager service, since the algorithm for navigating within the sankey chart is rather similar to that in the partition map and dashboard.